dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -120,7 +120,7 @@
     %br/
     %br/
 
-%script{src: minifiable_asset_path('js/signup.js')}
+%script{src: minifiable_asset_path('js/devise/registrations/new.js')}
```

Comment:
I hope we have test coverage for signin :)

I will probably ask for help with validation since I'm not sure how to get to all the places where these are used.